projects/templates/projects/project_detail.html

```diff
@@ -0,0 +1,55 @@
+{% extends "codeeve/index.html" %}
+
+{% block header %}
+{% endblock %}
+
+{% block content %}
+    <div class="content-section-a">
+
+        <div class="container">
+            <div class="row">
+                <a href="/projects/">Go to list all projects</a>
+                <a href="update/">Update the project</a>
+                <a href="delete/">Delete the project</a>
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">{{ project.title }}</h2>
+                    <p>{{ project.created_at|date:"D d M Y"  }}</p>
+                    <div
+                      {% if project.difficulty == "beginner" %}
+                         class = glossy-beginner
+                      {% elif project.difficulty == "intermediate" %}
+                         class = glossy-intermediate
+                      {% elif project.difficulty == "advanced" %}
+                         class = glossy-advanced
+                      {% endif %}>
+                        <p> {{ project.difficulty }}</p>
+                    </div>
+                    <p>{{ project.description}}</p>
+                    {% if project.max_members > project.participants.count %}
+                       <button type="button"> Join!</button>
+                    {% endif %}
+                  <!--  <p class="lead">Check out <a href="projects/">ongoing projects</a>
```

Comment:
I didn't put it there and wasn't sure if we will need it in the future, but I will delete it, we can just re-add it in the future.